unison-cli/src/Unison/Codebase/Editor/SlurpComponent.hs

```diff
@@ -1,88 +1,102 @@
-{- ORMOLU_DISABLE -} -- Remove this when the file is ready to be auto-formatted
 {-# LANGUAGE PatternSynonyms #-}
 
 module Unison.Codebase.Editor.SlurpComponent where
 
-import Unison.Prelude
-
-import Data.Tuple (swap)
-import Unison.Reference ( Reference )
-import Unison.UnisonFile (TypecheckedUnisonFile)
 import qualified Data.Map as Map
 import qualified Data.Set as Set
+import Data.Tuple (swap)
 import qualified Unison.DataDeclaration as DD
+import Unison.Prelude
+import Unison.Reference (Reference)
 import qualified Unison.Term as Term
+import Unison.UnisonFile (TypecheckedUnisonFile)
 import qualified Unison.UnisonFile as UF
 
-data SlurpComponent v =
-  SlurpComponent { types :: Set v, terms :: Set v }
-  deriving (Eq,Ord,Show)
+data SlurpComponent v = SlurpComponent {types :: Set v, terms :: Set v}
+  deriving (Eq, Ord, Show)
 
 isEmpty :: SlurpComponent v -> Bool
 isEmpty sc = Set.null (types sc) && Set.null (terms sc)
 
 empty :: Ord v => SlurpComponent v
-empty = SlurpComponent mempty mempty
+empty = SlurpComponent {types = mempty, terms = mempty}
```

Comment:
I also explicitly labelled all the other record usages to avoid the same error happening anywhere else.